apidoc/Titanium/UI/iOS/FeedbackGenerator.yml

```diff
@@ -44,6 +54,13 @@ properties:
     constants: Titanium.UI.iOS.FEEDBACK_GENERATOR_TYPE_*
     availability: creation
     optional: false
+  - name: style
+    type: Number
+    summary: The style of the feedback generator you want to create. This property is only required when using <Titanium.UI.iOS.FEEDBACK_GENERATOR_TYPE_IMPACT>.
+    constants: Titanium.UI.iOS.FEEDBACK_GENERATOR_STYLE_*
```

Comment:
I think this should be `FEEDBACK_GENERATOR_IMPACT_STYLE_*`, which is why the apidoc validator is complaining.

Reply:
Yep indeed! Let me fix that